stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetActivity.kt

```diff
@@ -193,6 +193,7 @@ internal class PaymentSheetActivity : BasePaymentSheetActivity<PaymentResult>()
         }
 
         viewBinding.toolbar.action.observe(this) { action ->
+            @Suppress("WHEN_ENUM_CAN_BE_NULL_IN_JAVA")
```

Comment:
Cleanup so commit on save doesn't warn about warnings.

Reply:
Nit: I'd prefer to add a no-op `null` or `else` clause to the `when` to resolve this warning